cmd/bootstrap/transit/version.go

```diff
@@ -0,0 +1,33 @@
+// Package build contains information about the build that injected at build-time.
+//
+// To use this package, simply import it in your program, then add build
+// arguments like the following:
+//
+//   go build -ldflags "-X github.com/dapperlabs/flow-go/version.semver=v1.0.0"
+package main
```

Comment:
I would move this to a separate package so it can be used for multiple executables without duplication.